src/layouts/about.jsx

```diff
@@ -2,7 +2,186 @@ import React from 'react';
 
 const About = () => (
   <div className="about">
-    <h2>About us!</h2>
+    <div className="group-photo-container text-center">
+      <img className="img-responsive group-photo" src="/images/about-cover.png" alt="A group discussion about Signsfive" />
+    </div>
+
+
+    <div className="container">
+      <section className="container py-5 px-5">
+        <h3 className="tracked pb-2">About SignsFive</h3>
+        <p>
+          <strong>
+            Make the search and share for STE(A)M (Science, Technology,
+            Engineering, <em>Art</em>, Math) signs easier.
+          </strong>
+        </p>
+        <p>
+          In fields where new technologies and technical terms are
+        always emerging, SignsFive will be a place where d/Deaf/HoH,
+        interpreters and ASL students (and hearing users of ASL) can
+        find signs for STE(A)M terms and contribute to the growing collection.
+        </p>
+      </section>
+      <div className="row py-5">
+        {/* <!--about-our-story--> */}
```

Comment:
`about-our-story` can be broken in their own component (jsx)

Reply:
done